meta/sai_serialize.h

```diff
@@ -12,13 +12,22 @@ extern "C" {
 #include <streambuf>
 #include <iomanip>
 #include <map>
+#include <unordered_map>
 #include <tuple>
 #include <cstring>
 
 #include "swss/logger.h"
 
 #include "sairedis.h"
 
+const std::unordered_map<sai_link_event_damping_algorithm_t, std::string> sai_link_event_damping_algorithm_to_name_map = {
+        {SAI_LINK_EVENT_DAMPING_ALGORITHM_DISABLED, "LINK_EVENT_DAMPING_ALGORITHM_DISABLED"},
+        {SAI_LINK_EVENT_DAMPING_ALGORITHM_AIED, "LINK_EVENT_DAMPING_ALGORITHM_AIED"}};
+
+const std::unordered_map<sai_redis_port_attr_t, std::string> sai_redis_port_attr_to_name_map = {
+        {SAI_REDIS_PORT_ATTR_LINK_EVENT_DAMPING_ALGORITHM, "LINK_EVENT_DAMPING_ALGORITHM"},
+        {SAI_REDIS_PORT_ATTR_LINK_EVENT_DAMPING_ALGO_AIED_CONFIG, "LINK_EVENT_DAMPING_ALGO_AIED_CONFIG"}};
+
```

Comment:
make this extern, and declare actual code in cpp file

Reply:
LINK_EVENT_DAMPING_ALGORITHM should be named in full name SAI_REDIS_PORT_ATTR_LINK_EVENT_DAMPING_ALGORITHM

Reply:
others too

Reply:
done